standard/lpdb/count.lua

```diff
@@ -0,0 +1,188 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Count
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local String = require('Module:StringUtils')
+local Team = require('Module:Team')
+
+local Condition = require('Module:Condition')
+local ConditionTree = Condition.Tree
+local ConditionNode = Condition.Node
+local Comparator = Condition.Comparator
+local BooleanOperator = Condition.BooleanOperator
+local ColumnName = Condition.ColumnName
+
+local Count = {}
+
+
+function Count.games(args)
+	args = args or {}
+
+	local lpdbConditions = Count._baseConditions(args)
+
+	return Count._query('game', lpdbConditions)
+end
+
+function Count.matches(args)
+	args = args or {}
+
+	local lpdbConditions = Count._baseConditions(args)
+	lpdbConditions = Count._tierConditions(args, lpdbConditions)
+
+	return Count._query('match', lpdbConditions)
+end
+
+function Count.tournaments(args)
+	args = args or {}
+
+	local lpdbConditions = Count._baseConditions(args, true)
+	lpdbConditions = Count._tierConditions(args, lpdbConditions)
+
+	return Count._query('tournament', lpdbConditions)
+end
+
+function Count.placements(args)
```

Comment:
check if smash (or smash-like wikis) uses the placements function of the old module
if yes —> add an overwrite for smash-like wikis (maybe Count/Legacy)

Reply:
Smash doesn't use it atm, what are the other smash like wikis?

Reply:
fighters and brawlhalla

Reply:
Not currently used on either